app/src/main/java/org/oppia/android/app/databinding/ImageViewBindingAdapters.java

```diff
@@ -43,6 +44,29 @@ public static void setImageDrawable(
     imageView.setImageResource(drawableResourceId);
   }
 
+  /**
+   * Binds a drawable indicated by {@link ChapterPlayState} to an {@link ImageView} via
+   * "android:src".
+   * Reference: https://stackoverflow.com/a/35809319/3689782.
+   */
+  @BindingAdapter("android:src")
```

Comment:
Could you maybe customize this so that it's not overloading android;src? I actually have a PR coming that's prohibiting using android:src for drawables, and this isn't really the same meaning of android:src, anyway. Suggest instead: playState:image (see profile:src for an example on how to use a custom namespace).

Also, please update the binding method name, too (e.g. to setPlayStateDrawable or similar).